<?xml version="1.0" ?>
<launch>

<param name="/use_sim_time"  value ="true"/>
<node pkg="depth_map" type="vio_depth" name="vio_depth" output="screen"/>
<!--node pkg="depth_map" type="loop_closure" name="loop_closure" output="screen"/-->
<arg name="path1" default="/home/edward/catkin_ws/src/posegraphcode/depthmapVINSxtion/data/warehat_passage_withpath"/>
<arg name="path2" default="/home/edward/catkin_ws/src/posegraphcode/depthmapVINSxtion/data/warehat_passage"/>
<arg name="path3" default="/home/edward/catkin_ws/src/posegraphcode/depthmapVINSxtion/data/109room_liuchengdata"/>
<arg name="path4" default="/home/edward/catkin_ws/src/posegraphcode/depthmapVINSxtion/data/velodyne"/>
<arg name="path5" default="/home/edward/catkin_ws/src/posegraphcode/depthmapVINSxtion/data/111room"/>
<arg name="file1" default="wearhat"/>
<arg name="file2" default="4"/>
<arg name="file3" default="loopclosure"/>
<arg name="file4" default="109room2"/>
<arg name="file5" default="velodyne"/>
<arg name="file6" default="111"/>
<node pkg="rosbag" type="play" name="rosbag" args="-r 1 -s 0 $(arg path5)/$(arg file6).bag --clock"/>
 <!--node pkg="tf" type="static_transform_publisher" name="velodyne" args="1 0 0 -1.57 0 -1.57 velodyne camera 100" /-->  
  <remap from="/camera1/depth/points" to="/camera/depth/points_filter"/>
 <!--node pkg="tf" type="static_transform_publisher" name="velodyne2" args="1 0 0 -1.57 0 0 camera camera2 100" /-->  
 <node pkg="rviz" type="rviz" name="rviz" args="-d $(find depth_map)/launch/vins_rviz_config.rviz"/>

</launch>
